src/lib/containers/query_wrapper_plot_nav/QueryWrapperPlotNav.tsx

```diff
@@ -100,23 +99,21 @@ const QueryWrapperPlotNav: React.FunctionComponent<QueryWrapperPlotNavProps> = p
       <SynapseContextConsumer>
         {context => (
           <QueryWrapper
-            {...rest}
+            {...props}
```

Comment:
Props that were destructured above weren't propagating (e.g. facetsToPlot, facetsToFilter). This bug has existed since this file was created, but I think the particular props and children that were affected have varied over time